pkg/kritis/crd/vulnzsigningpolicy/vulnzsigningpolicy.go

```diff
@@ -45,11 +46,13 @@ func ValidateVulnzSigningPolicy(vsp v1beta1.VulnzSigningPolicy, image string, vu
 
 	maxSev := vsp.Spec.ImageVulnerabilityRequirements.MaximumFixableSeverity
 	if maxSev == "" {
+		glog.Info("maximumFixableSeverity is unset, default to Critical.")
 		maxSev = constants.Critical
```

Comment:
Isn't this equivalent to AllowAll?

Reply:
Essentially yes with the tiny difference that `AllowAll` will also pass unknown severity levels. The values are inherited from image security policy, and I was also thinking if they are redundant.